prac_06/languages.py

```diff
@@ -0,0 +1,22 @@
+"""
+Languages
+Estimated: 30 minutes
+Actual: 24 minutes
+"""
+
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+
+programming_languages = [ProgrammingLanguage("Python", "Dynamic", True, 1991),
+                         ProgrammingLanguage("Ruby", "Dynamic", True, 1995),
+                         ProgrammingLanguage("Visual Basic", "Static", False, 1991)]
```

Comment:
Maybe you can ass in python, ruby and visual_basic objects that you've already created. DRY principal and a maintenance burden?